zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -121,4 +127,36 @@ private CompletableFuture<ResponseEntity<Object>> updateUserTask(
                 .withAuthentication(RequestMapper.getAuthentication())
                 .updateUserTask(request.userTaskKey(), request.changeset(), request.action()));
   }
+
+  @PostMapping(
+      path = "/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<Object> searchUserTasks(
```

Comment:
❌ We need to move this search functionality into a separate `UserTaskQueryController` class that is annotated with `@CamundaRestQueryController` so we can disable it by default for 8.6.